sos/plugins/virsh.py

```diff
@@ -21,8 +21,10 @@ class LibvirtClient(Plugin, RedHatPlugin, UbuntuPlugin, DebianPlugin):
     def setup(self):
         # virt-manager logs
         if not self.get_option("all_logs"):
+            self.add_copy_spec("/root/.cache/virt-manager/*", sizelimit=5)
             self.add_copy_spec("/root/.virt-manager/*", sizelimit=5)
         else:
+            self.add_copy_spec("/root/.cache/virt-manager/*")
```

Comment:
There are a couple of points here:

  * since you're just adding a new path, with otherwise identical arguments please use a single `add_copy_spec()` call and pass a list, i.e.:

```python
        self.add_copy_spec([
            "/root/.cache/virt-manager/*",
            "/root/.virt-manager/*"
        ], sizelimit=5)
```

  * Since you're collecting the same list of paths in both calls you could also use a local variable for the list to avoid repeating it in both `add_copy_spec()` calls.

More generally (these issues existed before your patch, but if we're changing this we should consider them):

  * The same set of logs is collected with `--all-log` and without - only the size limit is different. Normally `--all-logs` signals that rotated or other non-default log locations should be collected.

  * The plugin hard-codes a log size limit. Normally this should be left unset, so that the global `--log-size` option takes effect. If there is some special reason for a different value in this plugin it should be mentioned in the comments.

Reply:
Thanks, I see.

Reply:
Unfortunately, I have no special reason for uniquely setting "sizelimit=5"
in virsh plugin, and I plan to reflect the former comment only about 
the way to call add_copy_spec(). Is that ok?

Reply:
If there's no reason documented for having it then let's get rid of it and simplify the plugin - if there's no additional data to collect with `--all-logs` then that's fine. We can remove that branch and just allow the automatic size limits to apply.

Reply:
Thanks, I pushed patch v3 based on your comment.
Also, referring to other plugins (e.g. libvirt),
I change the file name of collecting virt-manager logs
from "*" to "*.log" to collect them more precisely.